tests/test_mle.py

```diff
@@ -177,6 +177,46 @@ def test_infer_node_dgs_w_error():
         assert res.rvalue > 0.9
 
 
+def test_infer_node_vals_and_errs_networkx():
+    edge_noise_stddev = np.random.rand()
+    g = generate_random_valid_regular_graph()
+    g = nx.convert_node_labels_to_integers(g)
```

Comment:
Is this important to the test? In practice the nodes will be string labels

Reply:
This is a bit misleading since the test cases already have integer labels (the reference function does not support string labels). The call to `nx.convert_node_labels_to_integers` is used to make the node labels coincide with their indices (i.e. number them consecutively), which simplifies the implementation of the test.

Reply:
Should we convert the labels to strings and handle string -> index conversion in the function then?

Reply:
Sorry, not sure I understand. This test checks for consistency with the existing `infer_node_values_and_errs`, which doesn't accept string labels. We could add a separate test to verify that string labels are handled correctly (but doing conversion from string labels to indices in the test function seems like it would replicate a large part of the functionality we're testing in the test implementation).

Reply:
In practice the nodes will identified by strings rather than integers. Want to make sure that this is tested, or explicit that the node labels need to be integers.

Reply:
OK, I think I've come up with a good way to test this (check that arbitrary relabeling of the nodes doesn't affect the result) in https://github.com/proteneer/timemachine/pull/987/commits/8f1df0e971e29f9b9e3ab51abe29bfe2616db5c0